crates/proof-of-sql/src/sql/ast/provable_expr_plan.rs

```diff
@@ -34,6 +35,10 @@ pub enum ProvableExprPlan<C: Commitment> {
     Equals(EqualsExpr<C>),
     /// Provable AST expression for an inequality expression
     Inequality(InequalityExpr<C>),
+    /// Provable numeric `+` expression
+    Add(AddSubtractExpr<C>),
+    /// Provable numeric `-` expression
+    Subtract(AddSubtractExpr<C>),
```

Comment:
NIT:
I don't think this needs to be separate variants.
If you want separate variants, I would suggest making `is_subtract` a const generic instead and then putting the appropriate const in the variant.
As it is, it's possible to have an Add variant that acts as a Subtract.